src/prefect/cli/deploy.py

```diff
@@ -864,17 +863,17 @@ async def _run_single_deploy(
 
 
 async def _version_info_from_options(
-    options: dict[str, Any], deploy_config: dict[str, Any]
+    client: "PrefectClient",
```

Comment:
This looks like it's unused
```suggestion
```

Reply:
Woops, good eye, that was from when I thought I needed to make the `client.server_type` check for Cloud.